host/xtest/pkcs11_1000.c

```diff
@@ -8343,6 +8343,7 @@ static CK_RV test_rsa_aes_wrap(ADBG_Case_t *c, CK_SESSION_HANDLE session,
 	return rv;
 }
 
+#ifndef CFG_CRYPTO_SE05X
```

Comment:
I don't think this should appear like that in the test code, because SE05X should be an implementation detail. Either there should be a software fallback (preferred), or we accept that not all implementations support this feature in which case we just accept `TEE_ERROR_NOT_IMPLEMENTED` as a success case.
@ldts what do you think?

Reply:
Right, the thing is that it doesn't make a lot of sense IMO to wrap an RSA private key when the secure element is enabled since what is being passed back to criptoki is not a valid RSA private key but a 64bit handle + the public components.

I haven't had the time to investigate why it fails yet; I guess it could be mbedtls trying to parse what it expects to be a private key and fails at it? But I agree with you, rather than just disabling the test, the test should accept a not implemented (or not supported even better) as a pass. not sure how hard doing it will be?

Reply:
Also thinking about it, it would NOT be right to disable it in xtests as some Secure Elements will in fact not complain (ie the NXP SE050-E, which doesnt support RSA in the element and does it in software instead).

So you are right, it should not be disabled here.